test/test-search.ts

```diff
@@ -63,9 +70,8 @@ test(
     async t => {
       const content = 'const a = require(\'h\' + \'t\' + \'t\' + \'p\');';
       const result = await search.search(content);
-      if (assert(result.dynamicEvals[0], 'position does not exist in array')) {
-        t.deepEqual(result.dynamicEvals[0].lineStart, 1);
-      }
+      t.true(!!result.dynamicEvals);
```

Comment:
I think TS can automatically determine that `result.dynamicEvals` is non-null here (and elsewhere), right?